docs/_includes/date-picker.html

```diff
@@ -0,0 +1,144 @@
+        <div class="date-picker__calendar-cell">
+            <!-- place calendar module here, no changes necessary -->
+        </div>
+    </div>
+</div>
+    {% endhighlight %}
+
+    <h3>Double Pane</h3>
+
+    <div class="date-picker" aria-label="date picker">
+        <div class="date-picker__header">
+            <div class="date-picker__header--inner">
+                <button
+                    class="icon-btn icon-btn--small icon-btn--transparent"
+                    aria-label="Previous Month - July 2019"
+                    variant="icon"
+                >
+                    <svg aria-hidden="true" class="icon icon--chevron-left" focusable="false">
+                        {% include symbol.html name="chevron-left" %}
+                    </svg>
+                </button>
+                <h3>August 2019</h3>
+                <h3>September 2019</h3>
+                <button
+                    class="icon-btn icon-btn--small icon-btn--transparent"
+                    aria-label="Next Month - September 2019"
+                    variant="icon"
+                >
+                    <svg aria-hidden="true" class="icon icon--chevron-right" focusable="false">
+                        {% include symbol.html name="chevron-right" %}
+                    </svg>
+                </button>
+            </div>
+        </div>
+        <div class="date-picker__body">
+            <div class="date-picker__calendar-cell">
+                {% include calendar-month.html dynamic="true" index="1" %}
+            </div>
+            <div class="date-picker__calendar-cell">
+                {% include calendar-month.html dynamic="true" index="2" %}
+            </div>
+        </div>
+    </div>
+    {% highlight html %}
+<div class="date-picker date-picker--single-month" aria-label="date picker">
+    <div class="date-picker__header">
+        <div class="date-picker__header--inner">
+            <button
+                class="icon-btn icon-btn--small icon-btn--transparent"
+                aria-label="Previous Month - July 2019"
+                variant="icon"
+            >
+                <svg aria-hidden="true" class="icon icon--chevron-left" focusable="false">
+                    <use href="#icon-chevron-left"></use>
+                </svg>
+            </button>
+            <h3>August 2019</h3>
+            <h3>September 2019</h3>
+            <button
+                class="icon-btn icon-btn--small icon-btn--transparent"
+                aria-label="Next Month - September 2019"
+                variant="icon"
+            >
+                <svg aria-hidden="true" class="icon icon--chevron-right" focusable="false">
+                    <use href="#icon-chevron-right"></use>
+                </svg>
+            </button>
+        </div>
+    </div>
+    <div class="date-picker__body">
+        <div class="date-picker__calendar-cell">
+            <!-- place calendar module here, no changes necessary -->
```

Comment:
One gap I see with the interactive example is that we can't see in the example of how buttons are marked up and placed inside of the table cells.